dojo/tools/anchore_grype/parser.py

```diff
@@ -23,44 +22,143 @@ def get_findings(self, file, test):
         data = json.load(file)
         dupes = dict()
         for item in data.get("matches", []):
-            cve = item["vulnerability"]["id"]
-            severity = self._convert_severity(item["vulnerability"]["severity"])
-            purl = PackageURL.from_string(item["artifact"]["purl"])
-            description = ""
-            description += f"\n**CVE:** {cve}"
-            if type(item["matchDetails"]) is dict:
-                description += f'\n**Matcher:** {item["matchDetails"]["matcher"]}'
+            vulnerability = item['vulnerability']
+            vuln_id = vulnerability["id"]
+            vuln_namespace = vulnerability.get('namespace', None)
+            vuln_datasource = vulnerability.get('dataSource', None)
```

Comment:
The method `get()` return `None` if the key is not in the dictionary.
So the correct form is:
```suggestion
            vuln_namespace = vulnerability.get('namespace')
            vuln_datasource = vulnerability.get('dataSource')
```

Reply:
done